initial RSA post (#1503)
oktadeveloper/okta-blog: _source/_posts/2024-05-01-okta-rsa.md

---
layout: blog_post
title: "Okta at RSA Conference 2024"
author: [michael-palermo, semona-igama, edunham]
by: advocate
communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
description: "Join Okta at RSA Conference on May 7, 2024"
tags: []
tweets:
- ""
- ""
- ""
image: 
type: awareness
---

> We are excited to have a presence at RSA Conference 2024! On Tuesday, May 7th, we will speak on [Identity: Your key to stop breaches](https://www.rsaconference.com/usa/agenda/session/Okta%20Identity%20Your%20key%20to%20stop%20breaches). Consider why you don't want to miss that session: 

Author: ```suggestion
We are excited to have a presence at RSA Conference 2024! On Tuesday, May 7th, we will speak on [Identity: Your key to stop breaches](https://www.rsaconference.com/usa/agenda/session/Okta%20Identity%20Your%20key%20to%20stop%20breaches). Consider why you don't want to miss this session: 
```

## Shared Signal Framework (SSF)
Does your current security solution wait until it sees suspicious activity in a critical application before logging an attacker out or prompting a legitimate user to re-authenticate? Chances are, accessing an essential application wasn't the first step an attacker took after compromising a user's account. In a genuinely zero-trust infrastructure, applications share information about user behavior that paints a holistic picture of whether an account's activity looks suspicious. Then each service responds appropriately for its threat model. 

The Shared Signals Framework (SSF) is a huge step in enabling formerly siloed applications to see the whole picture of whether a user's behavior looks normal or suspicious. SSF is developed by the OpenID Foundation, which is also the home of standards like OIDC, FAPI, and MODRNA. 

Although any software can send and receive these signals, Okta is uniquely positioned to work with our integrators to make SSF easy for identity administrators to use. 

> To learn more about the Shared Signals Framework, check out the OpenID foundation's [SSF Working Group](https://openid.net/specs/openid-sharedsignals-framework-1_0.html) or the explainer site [sharedsignals.guide](https://sharedsignals.guide/). Learn about how Okta and our integrators are deploying some of the world's first SSF implementations on [our developer podcast](TODO FIXME LINK)

Reviewer: ```suggestion
To learn more about the Shared Signals Framework, check out the OpenID foundation's [SSF Working Group](https://openid.net/specs/openid-sharedsignals-framework-1_0.html) or the explainer site [sharedsignals.guide](https://sharedsignals.guide/). 
```

## Universal Logout
What should happen next when suspicious activity is detected on a user's account? Identity providers can easily prevent a user from starting new sessions, but in the past, there hasn't been a good way to end all of a user's existing sessions. Nobody wants to re-authenticate for every task they perform, but in the past, these productivity and convenience features have given attackers an avenue to impersonate legitimate users by stealing session information. However, it's harder for an attacker to create a session than to continue a stolen session. 

Universal Logout addresses this problem by offering administrators and their automation a way to log users out of all their existing sessions with an identity provider. UL pairs with SSF and other intrusion detection tooling to get the best of both worlds: Users can keep the productivity benefits of long sessions and only do extra re-authentication when threats are detected, yet intruders experience the challenges of session termination immediately when they try to misuse stolen access. 

> While the [Universal Logout](https://developer.okta.com/docs/guides/oin-universal-logout-overview/) feature provided by Okta has not yet been released, you can build the endpoint that IdPs such as Okta or threat detection tools can call to end a user session ASAP by following the instructions on this blog: [How to Instantly Sign a User Out across All Your Apps](https://developer.okta.com/blog/2024/04/30/express-universal-logout).

Author: ```suggestion
While the [Universal Logout](https://developer.okta.com/docs/guides/oin-universal-logout-overview/) feature provided by Okta has not yet been released, you can build the endpoint that IdPs such as Okta or threat detection tools can call to end a user session ASAP by following the instructions on this blog: [How to Instantly Sign a User Out across All Your Apps](/blog/2024/04/30/express-universal-logout).
```

> If you would like to join a pilot program of integrators working with SSF and UL, contact us at dev-advocacy@okta.com.

Author: ```suggestion
If you want to join a pilot program of integrators working with SSF and UL, contact us at wic-dev-advocacy@okta.com.
```

The Shared Signals Framework (SSF) is a huge step in enabling formerly siloed applications to see the whole picture of whether a user's behavior looks normal or suspicious. SSF is developed by the OpenID Foundation, which is also the home of standards like OIDC, FAPI, and MODRNA. 

Although any software can send and receive these signals, Okta is uniquely positioned to work with our integrators to make SSF easy for identity administrators to use. 

To learn more about the Shared Signals Framework, check out the OpenID foundation's [SSF Working Group](https://openid.net/specs/openid-sharedsignals-framework-1_0.html) or the explainer site [sharedsignals.guide](https://sharedsignals.guide/). Learn about how Okta and our integrators are deploying some of the world's first SSF implementations on [our developer podcast](TODO FIXME LINK)

> > Abstract on SSF filled out by E. Dunham. Also please replace link to podcast.

Reviewer: ```suggestion
```

Reviewer: the above text is the abstract that I wrote in the google doc last week; is there more needed?

> > Link to blog pointing to podcast on SSF with Jamf

Reviewer: ```suggestion
```
podcast recording is happening today so I'll add the link to this post when I publish its post

Author: @edunham Would you mind changing this text then to remove references to linking? we shouldn't have a bad link.

## Universal Logout
What should happen next when suspicious activity is detected on a user's account? Identity providers can easily prevent a user from starting new sessions, but in the past, there hasn't been a good way to end all of a user's existing sessions. Nobody wants to re-authenticate for every task they perform, but in the past, these productivity and convenience features have given attackers an avenue to impersonate legitimate users by stealing session information. However, it's harder for an attacker to create a session than to continue a stolen session. 

Universal Logout addresses this problem by offering administrators and their automation a way to log users out of all their existing sessions with an identity provider. UL pairs with SSF and other intrusion detection tooling to get the best of both worlds: Users can keep the productivity benefits of long sessions and only do extra re-authentication when threats are detected, yet intruders experience the challenges of session termination immediately when they try to misuse ill-gotten access. 

> > Abstract on UL filled out by Semona Igama

Reviewer: ```suggestion
```

the above text *is* the abstract; we just need approval of it by @isemona

> Link to blog post on UL

> > Invitation for ISVs to contact us at dev-advocacy@okta.com if interested in testing w/Okta in a pilot program

Reviewer: ```suggestion
If you would like to join a pilot program of integrators working with SSF and UL, contact us at dev-advocacy@okta.com.
```
